### DISCUSSION TITLE
Can I use aliases plugin with wildcards to forward mail

### DISCUSSION THREAD

**Discussion by nogginbox** [haraka/Haraka#3188] [Q&A]
I have a few apps I sell and each has its own domain. I'd like to setup email forwarders for these domains that forward all email to my main email account.

I'm researching email servers to find a lightweight way to do this in a docker image on my server, and Haraka seems like a good choice?

If I set the mx records for a domain like mycoolapp.com and used this [aliases plugin](/haraka/Haraka/blob/master/docs/plugins/aliases.md) rule in Haraka:
```
{ "@mycoolapp.com" : { "action" : "alias", "to" : "app-support@mymaindomain.com" } }
```
Would all email sent to `any address @mycoolapp.com` be forwarded through to `app-support@mymaindomain.com`?

And if that is possible, would it also be possible to use a rule like this:
```
{ "*.support@mycoolapp.com" : { "action" : "alias", "to" : "app-support@mymaindomain.com" } }
```
where `*` is a wildcard?

**Comment by msimerson** [comment]
Try it and see. Seriously. And then report back here. (On my TODO list is to import my legacy aliases and use that plugin.)

**Reply by nogginbox** [reply]
I can't get it working, but I think logging may be broken in the alias plugin as I can't see any logs from the plugin after it logs that it has run the rcpt hook.

I'm going to have a go at debugging/fixing, but in the mean time I've logged a [bug about the logging](/haraka/haraka-plugin-aliases/issues/14).
